### PR TITLE
Specify accurate permission requirement in error message

### PR DESCRIPTION
**Pull request by dmohs**
No JIRA.

I received this helpful error message while trying to configure our server to use FireCloud dev and it saved me time, so I figured I'd reinvest that time into improving it.

We currently use FireCloud production and have given billing@firecloud.org the "Billing Account User" permission. For our usage, this seems to be working just fine. I'm proposing (via PR) that this is in fact all that is needed.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check that the **Product Owner** has signed off on any user-facing changes
- [x] **Submitter**: Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Database checks:
  * If PR includes new or changed db queries, include the explain plans in the description
  * Make sure liquibase is updated if appropriate
  * If doing a migration, take a backup of the
  [dev](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/backups?project=broad-dsde-dev&organizationId=548622027621)
  and
  [alpha](https://console.cloud.google.com/sql/instances/terraform-r4caezzc35c4tb7pgdhwkmme4y/backups?project=broad-dsde-alpha&organizationId=548622027621)
  DBs in Google Cloud Console
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] Tell your tech lead (TL) that the PR exists if they want to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via Slack and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala [hunk 1]
```diff
@@ -912,7 +912,7 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
             RequestComplete(StatusCodes.Created)
           }
         case None => Future.successful(RequestComplete(ErrorReport(StatusCodes.Forbidden, s"You must be a billing administrator of ${billingAccountName.value} to create a project with it.")))
```
Comment (dmohs) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala]: Poking around in this a bit more, I think this line could also say, "Billing Account User."

**Review by MatthewBemis — COMMENTED** [review]
there's a compilation error that seems to be related to the escaped quotes.

but the language change is a :+1: from me

**Review by MatthewBemis — APPROVED** [review]

**Comment by dmohs** [comment]
Doug visually inspected, so I'm going to merge since I think I have a good use case for testing this on dev once it's in.

**Comment by dmohs** [comment]
Just to follow-up, I've tested this change in multiple All of Us environments and it is definitely accurate. Yay.

**Pull request merged**